src/SignalR/server/SignalR/test/DefaultTransportFactoryTests.cs

```diff
@@ -103,7 +103,7 @@ public void DefaultTransportFactoryThrowsIfItCannotCreateRequestedTransport_Win7
             var transportFactory =
                 new DefaultTransportFactory(requestedTransport, loggerFactory: null, httpClient: new HttpClient(), httpConnectionOptions: null, accessTokenProvider: null);
             var ex = Assert.Throws<InvalidOperationException>(
-                () => transportFactory.CreateTransport(AllTransportTypes));
+                () => transportFactory.CreateTransport(AllTransportTypes, true));
```

Comment:
aside, in runtime repo the style requires naming the boolean parameter at the call site, ie ` () => transportFactory.CreateTransport(AllTransportTypes, useAck: true));`. this makes it more readable, IMO. looking at aspnetcore, it's inconsistent, about 50/50.